.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve Zephyr
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. mkdir build; cd build
+2. cmake -DBOARD=board_xyz
+3. make
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or console output**
+If applicable, add screenshots or console logs to help explain your problem.
```

Comment:
Change to:  

     If applicable, add a screenshot (drag-and-drop an image), or console logs 
     (cut-and-paste text and put a code fence (```) before and after, to help
     explain the issue.